docs/contributing/999-rfcs/2022-12-08-wing-inputs.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "#2726 Wing Inputs"
+description: "How to create, share, manage inputs"
+---
+
+# Wing Inputs
+- **Author(s)**: @Hasanaburayyan
+- **Submission Date**: {2023-12-08}
+- **Stage**: Draft
+
+Using Wing inputs in your application
+
+## Background
+
+Frequently applications have the need to read inputs, whether those inputs are sensitive data like passwords, connection strings, ssh keys or non-sensitive configuration such as retry counts, endpoint urls, etc..
+
+## Use cases
+
+- Sensitive Data Management
+	- Handling passwords, API keys, and secret tokens.
+	- Managing SSH keys for secure server access.
+	- Database connection strings.
+
+- Configuration Management
+	- Setting up environment-specific endpoint URLs (e.g., for different stages like development, staging, production).
+	- Configuring retry logic parameters such as retry counts and backoff strategies.
+	- Defining feature flags or toggle switches to enable/disable certain features dynamically.
+
+- Application Scaling
+	- Adjusting resource allocation parameters like CPU, memory limits, and autoscaling thresholds based on the environment.
+
+- Service Integrations
+	- Configuring webhook URLs and listener settings for external service events.
+
+- Security Policies
+	- Setting up access control lists or role-based access control parameters.
+
+- Network Configuration
+	- Specifying IP ranges, domain names, and port numbers for network settings.
+	- Configuring API gateway endpoints and routing rules.
+
+- Monitoring and Logging
+	- Setting up log levels, log retention policies, and monitoring thresholds.
+	- Configuring alerting rules and notification settings.
+
+- Deployment and Continuous Integration
+	- Specifying deployment targets and versioning details.
+	- Configuring build and test parameters for CI/CD pipelines.
+- Data Processing and Analytics
+	- Defining data retention policies and archival settings.
+	- Specifying parameters for data processing jobs
+
+-  Miscellaneous Settings
```

Comment:
+1 on Chris's suggestion to organize these based on their "type". 

This list is useful as anecdotal examples but it would be useful to try and categorize it using some higher level mental model because it's by definition non exhaustive.